app.js

```diff
@@ -1,23 +1,45 @@
 'use strict';
 
-function CookieStore (name, minCustomers, maxCustomers, avgCookies, hourlyCookieCount) { // this is a constructor function taking in 4 parameters.
+function CookieStore (name, minCustomers, maxCustomers, avgCookies, hourlyCookieSales) { // this is a constructor function taking in 4 parameters.
   this.name = name || 'Unknown'; // adding || (or) inputs a value if it doesn't meet true.
   this.minCustomers = minCustomers;  // the context of this is going to change. The keyword 'this' knows what it represents by the surrounding context
   this.maxCustomers = maxCustomers; // argument on the right is going into the parameter in the function.
   this.avgCookies = avgCookies;
-  this.hourlyCookieCount = hourlyCookieCount || [];// assigning an empty array []
+  this.hourlyCookieSales = hourlyCookieSales || [];// assigning an empty array []
   this.totalCountForDay = 0;
-  this.businessHours = ['07am ', '08am ', '09am ', '010am ', '011am ', '12pm ', '01pm ', '02pm ', '03pm ', '04pm ', '05pm ','06pm ','07pm ', '08pm '];
+  this.businessHours = ['06am','07am ', '08am ', '09am ', '010am ', '011am ', '12pm ', '01pm ', '02pm ', '03pm ', '04pm ', '05pm ','06pm ','07pm ', '08pm ', 'Daily Location Total'];
```

Comment:
Can you think of a way to do this without adding this last element into the array? See how you handled this problem on line 38.